orchagent/pfcwdorch.cpp

```diff
@@ -282,26 +263,27 @@ void PfcWdOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port)
         }
 
         sai_object_id_t queueId = port.m_queue_ids[i];
+        string queueIdStr = sai_serialize_object_id(queueId);
+
+        // Store detection and restoration time for plugins
+        vector<FieldValueTuple> countersFieldValues;
+        countersFieldValues.emplace_back("PFC_WD_DETECTION_TIME", to_string(detectionTime * 1000));
```

Comment:
Can we also put "action" in the table?

Reply:
Detection and restoration time is written to DB for plugins so that they can use it as input.
I suspect that it is required for later use to know which action to apply. Maybe it would be better to move PfcWdQueueEntry to the base class and reuse it?

Reply:
It's a good idea to move PfcWdQueueEntry to base class. 
We need action for alert-only mode, in which case we need to continuously run detect_plugin even when queue is stormed. It's OK that we don't add it now but we definitely need it in future.